Bottom: d729525a1543b7ec4c0e09555a1b02ed6079e423
Top:    222ead9ee8d99697c8894619e6c37eb8884759d0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-01-26 22:17:41 +0000

Make --enable-gl default when building the GTK HID


---

diff --git a/configure.ac b/configure.ac
index 281d029..260b3da 100644
--- a/configure.ac
+++ b/configure.ac
@@ -393,8 +393,12 @@ fi
 AC_MSG_CHECKING([for whether to use GL drawing])
 AC_ARG_ENABLE([gl],
 [  --enable-gl           Enable GL drawing (with GTK HID)],
-[],[enable_gl=no])
-
+[],[
+	case " $with_gui " in
+		*\ gtk\ *) enable_gl=yes;;
+		* ) enable_gl=no;;
+	esac
+])
 AC_MSG_RESULT([$enable_gl])
 AM_CONDITIONAL(USE_GL, test x$enable_gl = xyes)
